pkgs/development/python-modules/tatsu/default.nix

```diff
@@ -1,31 +1,26 @@
 { lib, buildPythonPackage, fetchFromGitHub, pythonOlder
-, colorama, mypy, pyyaml, regex
-, dataclasses, typing
-, pytestrunner, pytest-mypy
+, colorama, regex
+, pytestrunner, pytestCheckHook, pytest-mypy
 }:
 
 buildPythonPackage rec {
-  pname = "TatSu";
-  version = "5.5.0";
+  pname = "tatsu";
```

Comment:
Out of curiosity: Should one always use lowercase / the module name here? (https://github.com/neogeny/TatSu/blob/aec01f6478a37c69612d09050d7228ca3c7141bd/setup.py#L5 and https://github.com/neogeny/TatSu/blob/aec01f6478a37c69612d09050d7228ca3c7141bd/setup.py#L19)

Reply:
Yes, see https://github.com/NixOS/nixpkgs/pull/116344.